cl/phase1/network/gossip_manager.go

```diff
@@ -84,6 +84,7 @@ func (g *GossipManager) onRecv(ctx context.Context, data *sentinel.GossipData, l
 			err = fmt.Errorf("%v", r)
 		}
 	}()
+	data.Data = common.CopyBytes(data.Data)
```

Comment:
copy and assign to itself?

Reply:
Add comments to explain it. below is the explainer:

```go
// Make a copy of the gossip data so that we the received data is not modified.
	// 1) When we publish and corrupt the data, the peers bans us.
	// 2) We decode the block wrong
	data = &sentinel.GossipData{
		Name:     data.Name,
		Peer:     data.Peer,
		SubnetId: data.SubnetId,
		Data:     common.CopyBytes(data.Data),
	}
```

Reply:
hmm, I feel make a full copy of this GossipData could be more readable.